the Persian translation of the narrative bot

lalehmb opened the pull request:
I haven't translated the quotes as well as the magic-8 options; they were more attractive this way!

I'm sure Persians know little bit of English! ;)

lalehmb committed:
the Persian translation of the narrative bot

I haven't translated the quotes as well as the magic-8 options; they were more attractive this way!

I'm sure Persians know little bit of English! ;)

lalehmb commented:
Hi.

May you accept the Persian translation? or if there's anything I need to do, let me know?
Laleh

tgxworld commented:
Thanks @lalehmb  for contributing :) Your translations will need to be typed from left to right in the file otherwise YAML will complain that it is an invalid format.

lalehmb committed:
clearing format in word-office

I couldn't understand if it still has rtl format or not. would you comment me if it still has the problem please?

lalehmb commented:
thanks, but I got a little bit confused! and I don't understand what you
mean.

Anyway, I cleared the format in a word document and committed, but I don't
see any changes to the file! :(

may you guide more?

On Mon, May 15, 2017 at 12:16 PM, Guo Xiang Tan <notifications@github.com>
wrote:

> Thanks @lalehmb <https://github.com/lalehmb> for contributing :) Your
> translations will need to be typed from left to right in the file otherwise
> YAML will complain that it is an invalid format.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/discourse/discourse-narrative-bot/pull/13#issuecomment-301401045>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF9aQNb_4KxbXgGpIzOyxjBwoX4kBVVlks5r6ALSgaJpZM4NZCnm>
> .
>



-- 
lol

tgxworld commented:
No worries. You can have a look at https://github.com/discourse/discourse/blob/master/config/locales/server.fa_IR.yml. 

The indentation of a YAML file is important.

lalehmb committed:
changing the indents

I hope this is the right file, would you mention me if not?

thanks in advance

tgxworld commented:
Hi @lalehmb sorry but we've merged the narrative bot into core. You'll have to contribute the translation via  Transifex. Sorry for the trouble xD https://meta.discourse.org/t/contribute-a-translation-to-discourse/14882

lalehmb commented:
hmmmm :( I should've noticed that myself! I just thought about the planned
release date, which was due to today!

isn't there really anyway to push these translations to the core
repository?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Wed, May 31, 2017 at 1:15 AM, Guo Xiang Tan <notifications@github.com>
wrote:

> Hi @lalehmb <https://github.com/lalehmb> sorry but we've merged the
> narrative bot into core. You'll have to contribute the translation via
> Transifex. Sorry for the trouble xD https://meta.discourse.org/t/
> contribute-a-translation-to-discourse/14882
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/discourse/discourse-narrative-bot/pull/13#issuecomment-305002951>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF9aQFWaEhYm4zYgXvLj1gMEj6gEvOraks5r_IAEgaJpZM4NZCnm>
> .
>



-- 
lol
